app/containers/Channels/ChannelCreate.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { setFormType } from 'reducers/form'
+import ChannelCreateForm from './ChannelCreateForm'
+
+class ChannelCreate extends React.Component {
+  static propTypes = {
+    setFormType: PropTypes.func.isRequired
+  }
+
+  render() {
+    const { setFormType, ...rest } = this.props
+
+    return (
+      <ChannelCreateForm
+        onCancel={() => setFormType('CHANNELS')}
+        onSubmit={() => setFormType('CHANNELS')}
```

Comment:
this can be extracted and bound in constructor to avoid redundant re-renders
`this.setFormType = this.setFormType.bind(this, 'CHANNELS')`

Reply:
Good call. Updated.